### DISCUSSION TITLE
Welche Software für Systemübersicht-Skizzen?

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#807] [General]
Hallo Leute,

mit wechen Programmen erstellt ihr die Übersichten wie in https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Builds-%26-Examples ??

**Comment by 3joern** [comment]
Ich mache sowas in Affinity Designer, Adobe Indesign oder Illustrator, selbst Paint sollte gehen. Ah es gibt noch [Inkscape](https://inkscape.org/de/) das ist kostenlos

**Comment by meisterlampe** [comment]
Ich nutze für sowas Libre Office Draw (oder wer mag Visio). Da kann man dann z.B. auch mal Dinge neu anordnen und die Verbinder bewegen sich mit. (man kann die "Klebepunkte" für Verbinder auch selbst festlegen. Auch z.B. an Bildern, die man reingezogen hat)

**Comment by ottelo9** [comment]
Für sowas ist Microsoft Visio 1A. Ich verwende sogar noch das uralte 2003 ^^

**Comment by swingstate** [comment]
draw.io - kostenlos
